customer-config/scan-and-go/sng.device-api-settings.v1.json

```diff
@@ -10,6 +10,12 @@
       "minimum": 0,
       "maximum": 100,
       "description": "The minimum battery level required to release a handheld scanner."
+    },
+    "scannerInactiveDaysBeforeRemoval": {
```

Comment:
please rename to `hhsInactiveDaysBeforeRemoval` to have consistence naming considering the other setting name which is `hhsMinimumBatteryLevelRequiredToRelease`